js-sdk/providing_static_data.mdx

```diff
@@ -120,3 +120,34 @@ If you use namespaces, it should look like this:
 > This is default behavior, it's possible to customize in [`BackendFetch`](./api/web_package/plugins#backendfetch) plugin.
 
 
+## Data fallbacks
+
+You can provide local data as a fallback for CDN, for cases when network request fails.
+
+```ts
+const tolgee = Tolgee()
+  .use(BackendFetch({
+    prefix: '<your content storage link>',
+    fallbackOnFail: true
+  }))
+  .use(BackendFetch()) // your local data
```

Comment:
It might be more obvious from the example if we added some dynamic imports here.